orca/producer_test.go

```diff
@@ -509,26 +519,26 @@ func (s) TestProducerMultipleListeners(t *testing.T) {
 	checkReports(2, 1, 0)
 
 	// Register listener 3 with a more frequent interval; stream is recreated
-	// with the lower interval after the next report is received.  The first
-	// report will go to all three listeners.
+	// with this interval after the next report is received.  The first report
+	// will go to all three listeners.
 	oobLis3.cleanup = orca.RegisterOOBListener(lao.sc, oobLis3, lisOpts3)
 	fake.respCh <- loadReportWant
 	checkReports(3, 2, 1)
 	awaitRequest(reportInterval3)
 
-	// A normal report should go to all three listeners.
+	// Another report without a change in listeners should go to all three listeners.
 	fake.respCh <- loadReportWant
 	checkReports(4, 3, 2)
 
-	// Stop listener 2.  This does not affect the interval.  The next update
-	// goes to listeners 1 and 3.
+	// Stop listener 2.  This does not affect the interval as listener 3 is
+	// still the shortest.  The next update goes to listeners 1 and 3.
```

Comment:
should go

Reply:
and still has the shortest interval

Reply:
This level of review isn't important.  The meaning of this comment is very clear, and it's an internal comment in a test, not in external documentation.

Reply:
Wouldn't hurt though. But I see what you're saying